### PR TITLE
docker: divide production docker file in two

### PR DESCRIPTION
**Pull request by otaviopace**
## Description

Now there are two dockerfiles for production, one for
the server, and other for the worker, each one starting
the container at `src/bin/server.js` or `src/bin/worker.js`.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel comfortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

this closes https://github.com/pagarme/ghostbusters/issues/149

### CODE REVIEW

**Comment by claytonsilva** [comment]
@otaviopace nessa pr voce ainda nao vai adaptar o projeto para a nova infra não né?

**Comment by otaviopace** [comment]
O objetivo desse PR é simplesmente criar dois `Dockerfile`s, um para o `worker` e outro para o `server`, como é descrito na issue: https://github.com/pagarme/ghostbusters/issues/149
:)

**Comment by claytonsilva** [comment]
show  então to revisando aqui

File: Dockerfile.production.server [hunk 1]
```diff
@@ -17,3 +17,5 @@ COPY src $HOME/superbowleto/src
 EXPOSE 3000
 
 USER quarterback
+
+CMD ["node" "src/bin/worker.js"]
```
Comment (claytonsilva) [inline Dockerfile.production.server]: o server usa o mesmo js do worker?

Comment (grvcoelho) [reply Dockerfile.production.server]: Nope, tá errado hahahaha

Good Catch!

Comment (otaviopace) [reply Dockerfile.production.server]: ooops. Corrigido :) :+1:

**Comment by otaviopace** [comment]
@grvcoelho gostei da ideia, já que o `tldr` é open source como o `superbowleto`, acho válido mantermos um padrão cross-project. Já ajustei conforme a sugestão :)

**Pull request merged**